UPGRADING.rst

```diff
@@ -11,6 +11,17 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#2802 Provision deployment by running make deploy only
+======================================================
+
+Run … ::
```

Comment:
This sounds like I have to do this to upgrade. I don't think that's the case. If this purely informational, it should be worded that way, or this upgrading entry should be removed.